tool/gravity/cli/config.go

```diff
@@ -286,7 +297,9 @@ func (i *InstallConfig) CheckAndSetDefaults() (err error) {
 	}
 	err = i.validateApplicationDir()
 	if err != nil {
-		return trace.Wrap(err)
+		return trace.Wrap(err, "failed to validate installer directory. "+
+			"Make sure you're running the installer from the directory with the contents "+
```

Comment:
"Make sure you're launching install from the unpacked cluster image directory." <-- IIRC we wanted to get rid of the "installer" terminology.

Also, do you actually have to launch it from that directory? The install command also accepts a path IIRC.

Reply:
It is the default mode of operation - not sure a user would bother specifying the path on a command line although I could hint at that in the message.